internal/scanner/scanner.go

```diff
@@ -57,26 +69,92 @@ func (s *Scanner) ScanNamespace(namespace string) error {
 
 	// TODO this is for debugging, it should be remove in future steps!
 	for _, resource := range auditableResources {
-		fmt.Println("Policies: ")
+		log.Debug().Msg("Policies: ")
 		for _, policy := range resource.Policies {
-			fmt.Println(policy.GetName())
+			url, err := s.resourcesFetcher.GetPolicyServerURLRunningPolicy(context.Background(), policy)
+			if err != nil {
+				log.Debug().Err(err).Msg("CANNOT GET POLICY SERVER URL")
+				continue
+			}
+			log.Debug().Msgf("POLICY SERVER URL: %s", url.String())
 		}
 		for _, resource := range resource.Resources {
-			fmt.Println(resource)
-			fmt.Println("........")
+			log.Debug().Msgf("Opa!")
```

Comment:
Opa? Does it mean hello in Brazilian? :)

Reply:
Ops, my secret debugging method has been discovered.